sdk/monitor/monitor-ingestion/test/public/logsIngestionClient.spec.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { LogsIngestionClient } from "../../src";
+import { AggregateUploadLogsError, LogsIngestionClient } from "../../src";
```

Comment:
Can we add tests for the following please?

- AbortController scenario
- `onError`/`errorCallback`

Reply:
added